Tools/Pipeline/Common/PipelineTypes.cs

```diff
@@ -426,9 +426,16 @@ private static void ResolveAssemblies(IEnumerable<string> assemblyPaths)
             {
                 try
                 {                    
-                    var a = Assembly.LoadFrom(path);
-                    var types = a.GetTypes();
-                    ProcessTypes(types);
+                    // Only load in Assemblies that haven't already been loaded in.
+                    var reflectedAss = Assembly.ReflectionOnlyLoadFrom(path);
+                    bool found = assemblies.Any(ass => ass.FullName == reflectedAss.FullName);
```

Comment:
Please don't abbreviate variable names where the abbreviation could be considered offensive.  Just use the full name `reflectedAssemblies` and `assembly`.